v8/material-theme/assets/css/theme.css

```diff
@@ -282,6 +282,10 @@ img.thumbnail {
     padding: 0px;
 }
 
+img {
+    width: 100%;
```

Comment:
Making all images 100% wide will break all sites using this theme. Myself, I’d prefer it to be opt-in with a class, that does `max-width: 100%`.

Reply:
There is already an opt-in class, `thumbnail` available with that property. For my case, setting image attribute in org-mode using:
`#+ATTR_HTML: :class thumbnail`
set the class for image properly and images are now responsive.